src/addons/Select/Select.js

```diff
@@ -9,7 +9,7 @@ import Dropdown from '../../modules/Dropdown/Dropdown'
  * A <Select /> is sugar for <Dropdown selection />.
  * @see Dropdown
  */
-const Select = ({ className, ...rest }) => {
+function Select({ className, ...rest }) {
```

Comment:
This issue was raised and fixed.  Automated conformance testing depends on the constructor name.  Arrow functions are anonymous.  Added a conformance check to ensure all components have names.  Updated all stateless functional components to have function names.